contracts/security/src/storage/security.rs

```diff
@@ -0,0 +1,388 @@
+use cosmwasm_std::{Empty, Storage, Uint64};
+use cw_storage_plus::{Bound, Map};
+
+use crate::core::{
+    constants::{DEFAULT_SECURITY_TYPES_LIMIT, MAX_SECURITY_TYPES_LIMIT, SECURITY_TYPE_KEY},
+    error::ContractError,
+    msg::{Paginate, Security},
+};
+
+pub const SECURITY_TYPES: Map<(&str, &str), Empty> = Map::new(SECURITY_TYPE_KEY);
+
+/// Attempts to add a security to the list of acceptable types in the contract's storage.
+///
+/// # Arguments
+///
+/// * `storage` - A reference to the Deps' Storage object.
+/// * `security` - The security to add to the list of acceptable types.
+///
+/// # Examples
+/// ```
+/// add_type(deps.as_mut().storage, Security{category: "category".to_string(), name: None})?;
+/// `
```

Comment:
This doc annotation needs a triple tilde instead of a single to close code block. The other functions in this file need the same fix.

Reply:
Thanks, added.